stable/hackmd/templates/ingress.yaml

```diff
@@ -2,7 +2,7 @@
 {{- $fullName := include "hackmd.fullname" . -}}
 {{- $servicePort := .Values.service.port -}}
 {{- $ingressPath := .Values.ingress.path -}}
-apiVersion: apps/v1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Maybe capabilities can be used to make this work on both new and old clusters: https://helm.sh/docs/topics/chart_template_guide/builtin_objects/ ?

Reply:
Thanks, good idea :)
I just learned that helm create foobar already includes this in the basic ingress template, so I copied the solution from there.